crates/forge_analyzer/src/ir.rs

```diff
@@ -67,14 +70,15 @@ pub(crate) enum Terminator {
 
 // FIXME: ideally we should record the API call expression in the IR and the `UserFieldAccess` and `ApiCustomField` variants
 // should be removed and the type of the API call should be determined during dataflow.
-#[derive(Clone, Debug, Copy, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq)]
 pub enum Intrinsic {
-    Authorize,
+    Authorize(IntrinsicName),
     Fetch,
-    ApiCall,
-    SafeCall,
     UserFieldAccess,
     ApiCustomField,
+    ApiCall(IntrinsicName),
+    SafeCall(IntrinsicName),
+    JWTSign(PackageData),
```

Comment:
Can you rename this variant to `SecretFunction` or something more general? Also, don't forget to use VSCode's rename symbol function that's on `F2`.